.env.example

```
@@ -51,7 +51,23 @@ TELEGRAM_CHAT_ID="change-me-telegram-chat-id"
 # each). Any may be set; leave a value blank to disable that source.
 ROUTINES_BASE_URL="http://localhost:3000"
 ROUTINE_RSS_FEEDS="https://example.com/feed.xml,https://another.dev/rss"
-ROUTINE_DEVTO_TAGS="react,nextjs,typescript,javascript,webdev,frontend,css,architecture,designpatterns,cleancode,ai,machinelearning"
+# dev.to tags mapped to the blog's 5 editorial categories (ia, prompt, dev,
+# arch, ts) so curation stays on-topic.
+ROUTINE_DEVTO_TAGS="ai,machinelearning,llm,promptengineering,webdev,javascript,node,architecture,designpatterns,softwareengineering,cleancode,typescript,react,nextjs"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)